### PR TITLE
#8728 prototype

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #

### CODE REVIEW

File: packages/calcite-components/src/assets/styles/_sortable.scss [hunk 1]
```diff
@@ -1,7 +1,9 @@
 @mixin sortable-helper-classes() {
   .calcite-sortable--chosen,
   .calcite-sortable--ghost,
-  .calcite-sortable--drag {
+  .calcite-sortable--drag,
+  .calcite-sortable--fallback {
+    position: relative;
```
Comment (driskull) [inline packages/calcite-components/src/assets/styles/_sortable.scss:6]: this part fixes the styling error for the dragging element.

File: packages/calcite-components/src/components/list-item/list-item.scss [hunk 2]
```diff
@@ -2,6 +2,8 @@
   @apply flex flex-col;
   --calcite-list-item-icon-color: theme("colors.brand");
   --calcite-list-item-spacing-indent: theme("spacing.4");
+  // uncomment this and it works with `forceFallback: true,`
+  // padding-block: 10px;
```
Comment (driskull) [inline packages/calcite-components/src/components/list-item/list-item.scss:6]: Introducing a padding between items seems to fix the dragging when fallback is enabled. Need to figure out why dragging on a web component isnt triggering elements to swap unless there is space between them. cc @jcfranco

File: packages/calcite-components/src/utils/sortableComponent.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ export const CSS = {
   ghostClass: "calcite-sortable--ghost",
   chosenClass: "calcite-sortable--chosen",
   dragClass: "calcite-sortable--drag",
+  fallbackClass: "calcite-sortable--fallback",
```
Comment (driskull) [inline packages/calcite-components/src/utils/sortableComponent.ts:16]: Added fallback class css for usage if necessary.

**Comment by driskull** [comment]
@jcfranco I can get sortable working if I change the following:

https://github.com/SortableJS/Sortable/blob/07708fa1e7d9cf9d4563a7b1a09f7a47771522ed/src/Sortable.js#L764-L770


From:

```js
			if (parent) {
				do {
					if (parent[expando]) {
						let inserted;

						inserted = parent[expando]._onDragOver({
							clientX: touchEvt.clientX,
```

To:

```js
      if (parent) {
        do {
          // todo
          const test = parent[expando] || dragEl.parentNode[expando];
          if (test) {
            var inserted = void 0;
            inserted = test._onDragOver({
              clientX: touchEvt.clientX,
```

**Comment by jcfranco** [comment]
Thanks for working on this! ✨💪✨

Could we apply the fallback and extra padding only for mobile platforms (via [userAgent](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/utils/browser.ts#L13))? We'd have to note this as a known issue for the time being. cc @ashetland @geospatialem 

I was not able to find any similar issue in SortableJS, so this might be an upstream issue that they would need to address. Could you submit an issue for this? Sharing your [findings above](https://github.com/Esri/calcite-design-system/pull/8730#issuecomment-1942934093) would be helpful.

**Pull request closed without merging**